_layouts/default.html

```diff
@@ -8,7 +8,7 @@
     {% feed_meta %}
     <title>{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}</title>
     <link href="/css/normalize.min.css" rel="stylesheet">
-		<link href="/css/bootstrap.css" rel="stylesheet">
+		<link href="/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
this breaks the layout:

<img width="665" alt="image" src="https://user-images.githubusercontent.com/1359421/165200696-b368cfca-7e94-470b-aa45-2af164319123.png">

Reply:
Hmm, interesting. That hints that the site styles rely on a cascade from bootstrap that changes depending on minified/un-minified state. I'll investigate and see what I can dig up.

Reply:
Ah ha, the un-minified version of bootstrap is `v5.0.0-beta1`, while the minified one is `v3.3.5`, hence the layout breakage. I'll make a PR to bring the minified file to the same version as the un-minified.

Also looks like the current bootstrap version is 5.0.2. Any objection to updating out of beta?